nrf_security/doc/backend_config.rst

```diff
@@ -7,27 +7,25 @@ Backend configurations and supported features
    :local:
    :depth: 2
 
-This section covers the configurations available when one or more nrf_security backends are enabled.
-This includes linking directly to the backend library or utilizing the Mbed TLS glue layer.
+This section covers the configurations available when either linking directly to a backend library or when using the Mbed TLS glue layer.
 
 .. _nrf_security_backend_config_multiple:
 
 Configuring multiple backends
 *****************************
 
-Different backends can support different cryptographic algorithms.
+Different backends support different cryptographic algorithms.
 The configuration options listed in subsequent sections are either *Glue*, *Shared*, or *Choice*.
 
 Glue
-   The configuration options will list `Glue` for the cryptographic algorithms that can be enabled in multiple backends at the same time.
+   Can be enabled in multiple backends simultaneously.
 Shared
-   For some cryptographic features, the implementation is only available in the :ref:`nrf_security_backends_orig_mbedtls`.
-   For convenience, this is made available in any backend, although the implementation will always use open-source code from the Arm Mbed TLS project.
+   Is only supported by the :ref:`nrf_security_backends_orig_mbedtls` and can be enabled even though the backend has been disabled.
 Choice
-   The configuration options will list `Choice` for the cryptographic algorithms that are supported by multiple backends, but only one of them can be enabled at the same time.
-   To enable the cryptographic algorithm, a `base` configuration must be enabled, and then the backend can be selected using a setting prefixed with ``CONFIG_CHOICE_``.
+   Can not be enabled in multiple backends simultaneously. A choice must be made.
```

Comment:
I am hesitant on removing the statement on 'Choice" referencing the table content

Reply:
TODO